src/scripts.js

```diff
@@ -12,42 +12,42 @@ import './images/turing-logo.png'
 
 import UserRepository from './UserRepository';
 import Hydration from '../src/Hydration';
-import Sleep from '../src/Sleep';
+// Sleep was unused
 import User from './User';
 import apiCalls from './apiCalls';
 import { Chart } from 'chart.js/auto';
 
 // Query Selectors
-let userInfo = document.querySelector('.user-info')
-let stepGoalDisplay = document.querySelector('.step-goal')
-let friendsListDisplay = document.querySelector('friends-list')
-let hydroGraph = document.getElementById("weekWater")
-let todayWater = document.querySelector(".today-water")
-let weeklyWaterContainer = document.querySelector(".weekly-water-container")
+let userInfo = document.querySelector('.user-info');
+let stepGoalDisplay = document.querySelector('.step-goal');
+// Removed two unused lines here
+let todayWater = document.querySelector(".today-water");
+let weeklyWaterContainer = document.querySelector(".weekly-water-container");
+// Do all of these need to be `let`s? Most people I know use `const` unless you're actually going to reassign the value of the variable
 
 
 //Global Variables
 let allUserData = [];
-let user 
-let currentRepo 
-let hydration
-let hydrationData
-let sleep
-let userData
+let user;
```

Comment:
I would use more `consts`. A lot of these can just be set at the time you call the function they're in. For example, you could remove this line, change `getRandomUser()` to just `return currentRepo.userData[getRandomIndex(currentRepo.userData)];`,  and then instead of what you're doing now, when you call `getRandomUser()`, just do `const user = getRandomUser()` at that time.